service/src/test/java/uk/gov/hmcts/reform/fpl/service/DocumentGeneratorServiceTest.java

```diff
@@ -44,6 +44,9 @@ void shouldGenerateSubmittedFormDocumentWhenCaseIsFullyPopulated() throws IOExce
         String content = textContentOf(createServiceInstance(clock).generateSubmittedFormPDF(populatedCaseDetails(),
             Pair.of("userFullName", "Emma Taylor"))
         );
+
+        System.out.println(content);
```

Comment:
Remove logging please. But this was my fault as you where helping me with debugging PDF

Reply:
Done